### ISSUE TITLE
which directions are the x、y、z axises of the world coordinate system of ARCore?

### ISSUE THREAD

**Issue by Hotpotboy** [google-ar/arcore-android-sdk#770]
I just know the origin of the world coordinate system is the position when arcore started.

**Comment by lvonasek** [comment]
Y is direction sky. X and Z are every session different (depending on device rotation when ARCore starts).

You might have an interest in this modified example: https://github.com/inio/arcore-android-sdk/tree/unsupported-geo-oriented

**Comment by fredsa** [comment]
For advice and general questions, please use the `Sceneform` tag on
Stack Overflow: https://stackoverflow.com/questions/tagged/Sceneform

In addition, please see the Sceneform SDK [developer documentation](https://developers.google.com/ar/develop/java/sceneform/) and [API reference](https://developers.google.com/ar/reference/java/sceneform/reference/packages).

**Comment by FannyWF** [comment]
Is the arcore world coordinate system fixed? If it's fixed, does the mobile phone pose affect the world coordinate system?

According to my mobile phone test, I think the Y axis of arcore world coordinate system is vertical to the ground, X is facing the right along the mobile phone screen, and the origin is the device center of arcore session initialization. So in my opinion, arcore world coordinate system and geographic coordinate system only have an included angle of equipment direction. But I'm not sure if the initial attitude of the mobile phone will affect the world coordinate system.

I want to transfer the world coordinate system to the geographical coordinate system. But my conversion result is different from the actual position. I doubt I have made clear the definition of the world coordinate system in arcore.

I tried to ask stackflow, but no one answered me
